wordpress-mobile/gutenberg-mobile
# Aztec wrapper example upgrade android tools

We are not actively maintaining the Aztec Wrapper's example app anymore but, had to give it a shot so to test out #1196.

To test:

Follow the Aztec Wrapper's [README](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/react-native-aztec/README.md) to run the example app on Android. It should run.

Update release notes:
